sw/device/lib/dif/dif_rv_core_ibex.h

```diff
@@ -280,6 +280,19 @@ dif_result_t dif_rv_core_ibex_get_rnd_status(
     const dif_rv_core_ibex_t *rv_core_ibex,
     dif_rv_core_ibex_rnd_status_t *status);
 
+typedef uint32_t dif_rv_core_ibex_fpga_info_t;
+/**
+ * Read the FPGA build timestamp info. This function only return valid data for
+ * fpga, for all other variants it is simply 0.
+ *
+ * @param rv_core_ibex Handle.
+ * @param info Valid pointer to receive the FPGA timestamp info.
```

Comment:
Nit: is there any particular reason to specify that this must be a valid pointer? That seems like it should be an unstated default requirement for all parameters, unless otherwise indicated.

Reply:
changed.